pkg/ovs/openflow/utils.go

```diff
@@ -17,13 +17,15 @@ package openflow
 import (
 	"fmt"
 	"net"
+	"slices"
 	"strings"
 
+	"k8s.io/klog/v2"
+
```

Comment:
the original grouping is correct

Reply:
Not sure why?

Reply:
Aren't k8s.io/klog and antrea.io/libOpenflow are all third-party library imports? I believe we have been enforcing the best practice since the first day:
- Standard library imports
- Third-party library imports
- Project-specific (local) imports